sandbox.html

```diff
@@ -406,6 +408,15 @@
   </div>
 </template>
 
+<template data-action="setSelection">
+  <div>
+    <label for="layerPath">Path to the dataview layer * (GeoJSON Array)</label>
+    <textarea name="layerPath" id="layerPath" rows="5" required></textarea>
+    <label for="values">List of Object identifiers * (GeoJSON Array)</label>
+    <textarea name="values" id="values" rows="5" required></textarea>
+  </div>
```

Comment:
There should be a <div> around every form control and its label. There is some spacing defined between these containers.